.craft.yml

```
@@ -9,3 +9,10 @@ targets:
   - name: github
     tagPrefix: v
     access: public
+  - name: gcs
+    includeNames: /.*\.js.*$/
+    bucket: sentry-js-sdk
+    paths:
+      - path: /{{version}}/
```

```suggestion
      - path: /sentry-fullstory/{{version}}/
```

Just to be sure we are not messing up anything with conflicting versioning.